server/controllers/airports.js

```diff
@@ -0,0 +1,20 @@
+const Airport = require('../models').Airport;
+
+module.exports = {
+  create(req, res) {
+    return Airport
+      .create({
+        name: req.body.name,
```

Comment:
Same as above. We are allowed to create airports reports with `null` name and code